client/stylesheets/shared/_reset.scss

```diff
@@ -17,10 +17,13 @@
 		.wp-responsive-open #wpbody {
 			right: -14.5em;
 		}
-		#wpcontent,
-		#wpbody-content {
+		#wpcontent {
 			min-height: calc(100vh - #{$adminbar-height-mobile});
 		}
+
+		#wpbody-content {
+			min-height: 100vh;
```

Comment:
We probably only want to apply this on onboarding pages.  Most pages still having a padding on `#wpbody` that will make this longer than the viewport with no content.